eth/tools/fixtures/normalization.py

```diff
@@ -66,7 +66,10 @@ def normalize_int(value):
 
 
 def normalize_bytes(value):
-    if is_hex(value) or len(value) == 0:
+    # type(value) maybe also bytes. But for `is_hex(value)` function
+    # value has to be only of the type string
+    value = value.decode('utf-8')
+    if len(value) or is_hex(value) == 0:
```

Comment:
This isn't correct, the `== 0` didn't stay with the `len` check.

Probably best to update this to:

```python
if is_bytes(value):
    return b''
elif is_text(value) and is_hex(value):
    return decode_hex(value)
else:
    raise TypeError(...)
```

Reply:
Oops sry about that. Will make the changes ASAP